explorer/src/lib/url/makeNodeUrl.js

```diff
@@ -1,37 +1,40 @@
 /**
- * Constructs a node URL based on the subdomain
+ * Constructs a node URL based on the current subdomain
  *
  * @returns {URL} nodeUrl
  */
-function makeNodeUrl() {
+function makeNodeUrl(path = "") {
   const domains = window.location.hostname.split(".");
 
   let node;
 
   switch (domains[0]) {
     case "apps": // mainnet
       node = new URL(
-        `${window.location.protocol}nodes.${window.location.host}`
+        `${window.location.protocol}nodes.${window.location.host}${path}`
       );
       break;
     case "devnet":
       node = new URL(
-        `${window.location.protocol}devnet.nodes.${window.location.host}`
+        `${window.location.protocol}devnet.nodes.${window.location.host}${path}`
       );
       break;
     case "testnet":
       node = new URL(
-        `${window.location.protocol}testnet.nodes.${window.location.host}`
+        `${window.location.protocol}testnet.nodes.${window.location.host}${path}`
       );
       break;
     default: // localnet
       node = new URL(
-        `${import.meta.env.VITE_RUSK_PATH || "/"}`,
+        `${import.meta.env.VITE_RUSK_PATH || "/"}${path}`,
         import.meta.url
       );
 
       if (import.meta.env.VITE_NODE_URL) {
-        node = new URL(import.meta.env.VITE_NODE_URL, import.meta.url);
+        node = new URL(
```

Comment:
I see this is not part of this PR, but I don't get why you're redeclaring `node` here: shouldn't this be an "if / else" maybe?